From e6c3011f5dc9107fd8e3fdc1cbbcfaa4255fc22b Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Thu, 31 Jan 2013 09:02:51 +0000
Subject: [PATCH 0723/1083] thermal: rcar: use parenthesis on macro

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Zhang Rui <rui.zhang@intel.com>
(cherry picked from commit 9dde8f86085d283042718f88eed017eccad73ab9)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit a3bfd49f89b850f20818ca53dbdc1fe1797d167b)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/thermal/rcar_thermal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/thermal/rcar_thermal.c b/drivers/thermal/rcar_thermal.c
index aad892b..cedef22 100644
--- a/drivers/thermal/rcar_thermal.c
+++ b/drivers/thermal/rcar_thermal.c
@@ -47,7 +47,7 @@ struct rcar_thermal_priv {
 };
 
 #define MCELSIUS(temp)			((temp) * 1000)
-#define rcar_zone_to_priv(zone)		(zone->devdata)
+#define rcar_zone_to_priv(zone)		((zone)->devdata)
 
 /*
  *		basic functions
-- 
1.8.3.2

